assets/src/application/services/apollo/mutations/useMutationHandler.ts

```diff
@@ -25,7 +25,7 @@ const useMutationHandler: MutationHandler = (getMutationOptions) => {
 	const client = useApolloClient();
 	const [result, setResult] = useState(DEFAULT_RESULT);
 	const ifMounted = useIfMounted();
-	const toaster = useToaster();
+	const toaster = useSystemNotifications();
```

Comment:
for some reason the word system there makes me think of desktop notification( from OS notifications maybe)